features/features.go

```diff
@@ -728,6 +728,14 @@ var (
 					enableIn(configv1.DevPreviewNoUpgrade).
 					mustRegister()
 
+	_ = newFeatureGate("TestOCAdminUpgradeTechPreview").
+		reportProblemsToJiraComponent("oc / update").
+		contactPerson("wking").
+		productScope(ocpSpecific).
+		enhancementPR(legacyFeatureGateWithoutEnhancement).
```

Comment:
[`verify` is mad about the lack of enhancement][1], but I'm going to leave it alone for now, while I wait for API reviewers to decide how they feel about the general idea of a feature-gate to cover origin test-case enablement.

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_api/2337/pull-ci-openshift-api-master-verify/1925261883590316032#1:build-log.txt%3A8018